pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -130,17 +140,18 @@ type PullSource struct {
 // reference.
 func (r *Registry) PullSourcesFromReference(ref reference.Named) ([]PullSource, error) {
 	var endpoints []Endpoint
-
-	if r.MirrorByDigestOnly {
```

Comment:
This still needs to honor the `Registry.MirroByDigestOnly` flag, for compatibility with older config files and older OpenShift.

Reply:
I agree, see my comment in https://github.com/containers/image/pull/1411/files#r753042127